apps/src/lib/node/ledger/shell/init_chain.rs

```diff
@@ -21,12 +23,13 @@ where
     /// Create a new genesis for the chain with specified id. This includes
     /// 1. A set of initial users and tokens
     /// 2. Setting up the validity predicates for both users and tokens
-    /// 3. A matchmaker
+    /// 3. Validators
+    /// 4. The PoS system
+    /// 5. The Ethereum bridge parameters
```

Comment:
+1 for splitting up `init_chain`, think it was long overdue, though we should push this back to `main` as well

Reply:
I was hoping that the functions would secretly all be really similar and we could do something generic. Didn't seem to work out that way though :cry: